x/bridge/README.md

```diff
@@ -97,6 +93,18 @@ The transfer process is straightforward:
 * When the `x/bridge` module receives an inbound transfer, it starts accumulating votes for this transfer. Once the vote threshold is met, it employs the tokenfactory's [Mint](../tokenfactory/README.md#mint) method to mint the appropriate amount of coins for the destination address.
 * For an outbound transfer, the `x/bridge` module uses the tokenfactory's [Burn](../tokenfactory/README.md#burn) method to burn the specified amount of coins from the sender's address. Subsequently, an **EventOutboundTransfer** event is emitted and handled by the **super valset** through the **observer**.
 
+### Rate limiting
+
+**Rate limiting** is a strategy used to restrict the amount of incoming volume over a specified period of time. For instance, consider a scenario where the limit is set to 1 BTC every 10 minutes. Should the module receive more than 1 BTC within this 10-minute timeframe, any excess volume above the permitted amount would not be minted and instead be placed in **quarantine** (for more details, please see the [Quarantine](#quarantine) section). This approach is crucial for ensuring chain security, as it prevents the system from being overwhelmed by excessive, uncontrolled minting. Essentially, rate limiting acts as a safeguard, setting a maximum on potential losses the chain might incur due to unforeseen malicious activities.
```

Comment:
If this is the full section of rate limiting, I would touch on two types of rate limiting

1. Time based caps (i.e. only 5 BTC every day)
2. Single transaction limits (i.e. a single transaction can only take up 20 percent of the daily cap)

I would also use a more realistic time frame, our ibc rate limits are on the order of day and week

Reply:
We haven't discussed how single transaction limits should work in terms of quarantining. Should we put the remaining portion into the queue? I guess in that case, we need to check every transfer popping from the queue and then put it back if there is still a remainder after execution.

Thus, if the daily cap is 10 BTC and the transfer limit is 20%, then the transfer of 10 BTC will be completed within 5 days (10 * 0.2 = 2 BTC per day).